extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -491,8 +491,7 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 			title: loc.workerNodesInformation,
 			ariaLabel: loc.workerNodesInformation,
 			width: '15px',
-			height: '15px',
-			enabled: false
+			height: '15px'
```

Comment:
You should specify this as an informational button (buttonType == Informational) - right now I believe this is going to have a click handler attached which means screen readers will announce it as clickable. That's confusing for users since in this case clicking it doesn't do anything. Informational buttons shouldn't have the click handlers so will be read out correctly. 

(same for all the other informational buttons in the dashboards)

Reply:
button with type === information is clickable, it was introduced by Hale's ML dashboard work, it has image, subject and description. 

i believe info button here is 
![image](https://user-images.githubusercontent.com/13777222/127690615-add1b272-5480-46e7-9e52-b9488c48172d.png), in text component, you can provide a description, if provided, it will show an info icon next to it. you should leverage that.

Reply:
Huh...let me check that. Because that doesn't sound right - informational buttons typically don't have any action associated with them that I'm aware of.